packages/orval/src/utils/options.ts

```diff
@@ -201,6 +201,7 @@ export const normalizeOptions = async (
           useQuery: true,
           useMutation: true,
           signal: true,
+          shouldExportMutatorHooks: true,
```

Comment:
I wonder if for consistency it should be `useMutationHooks` ?

Reply:
Not sure because it's still generated but just not exported anymore